app/presenters/idv/in_person/verification_results_email_presenter.rb

```diff
@@ -6,7 +6,7 @@ class VerificationResultsEmailPresenter
       attr_reader :enrollment, :url_options
 
       # update to user's time zone when out of pilot
-      USPS_SERVER_TIMEZONE = ActiveSupport::TimeZone['America/New_York']
+      USPS_SERVER_TIMEZONE = (ActiveSupport::TimeZone['America/New_York']).freeze
```

Comment:
ditto about `.dup.freeze` for the shared object
```suggestion
      USPS_SERVER_TIMEZONE = ActiveSupport::TimeZone['America/New_York'].dup.freeze
```